### PR TITLE
65 remove hardcoded strings

### LINKED ISSUE #65: Remove hardcoded strings in view
- [x] donations view
- [x] recipients view
- [ ] category records view
- [x] Home
- [x] Top bar
- [x] `Back` and `Edit` links
- [x] `Destroy` links

There are a lot of strings that are hard-coded in, for example the "Name:" text on the show pages for a donation.

Replace these with calls for translated strings :) See 84cb5b6 for how I did some of this already

### PR DESCRIPTION
**Pull request by qubist**
closes #65

### CODE REVIEW

**Comment by qubist** [comment]
^ the "closes #n" thing worked with it in the description.

**Comment by qubist** [comment]
@lbraun Some of the category_records stuff is a little unclear to me. I basically didn't touch anything in the `views/category_records` folder unless I could find it through the UI.

File: config/locales/en.yml [hunk 1]
```diff
@@ -15,14 +15,48 @@ en:
...
     form:
       existing_categories: "Existing categories"
+    new:
+      title: "New category"
   defaults:
     destroy_confirm: "Are you sure?"
     destroy_link: "Destroy"
     edit_link: "Edit"
+    back_link: "Back"
     new_link: "New %{model}"
```
Comment (lbraun) [inline config/locales/en.yml:25]: You could use this syntax in more places to reduce duplication. For example, right now we have separate translations for "donations.new.title", "recipients.new.title", etc. but they are all quite similar.

Comment (lbraun) [reply config/locales/en.yml:25]: Same goes for edit.title

**Comment by lbraun** [comment]
> @lbraun Some of the category_records stuff is a little unclear to me. I basically didn't touch anything in the `views/category_records` folder unless I could find it through the UI.

Yeah, I think that whole feature needs a lot of work. I might do a bit on this today since it's so rainy here. You free?

**Comment by lbraun** [comment]
Merging since this looks good, feel free to make further improvements in a new PR if you want.

**Pull request merged**